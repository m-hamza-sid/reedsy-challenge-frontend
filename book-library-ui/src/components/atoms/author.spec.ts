import { mount } from "@vue/test-utils";
import { describe, expect, test } from "vitest";
import Author from "./author.vue";

describe("author", () => {
  test("should render with name prop", () => {
    const iName = "some name";
    const wrapper = mount(Author, { props: { name: iName } });
    expect(wrapper.text()).toContain(iName);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
