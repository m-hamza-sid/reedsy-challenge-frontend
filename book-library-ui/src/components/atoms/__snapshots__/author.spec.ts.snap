// Vitest Snapshot v1

exports[`author > should render 1`] = `"<div class=\\"text-left\\">some name</div>"`;

exports[`author > should render with name prop 1`] = `"<div class=\\"text-left\\">some name</div>"`;
